src/models/jobModels.ts

```diff
@@ -0,0 +1,23 @@
+import mongoose from "mongoose";
+
+export const jobModels = mongoose.model('jobform',
+    new mongoose.Schema({
+        title:{
+            type:String,
+            required:true
+        },
+        category: {
+            type: String,
+            enum: ["Frontend Positions", "Backend positions", "Full-stack positions", "Others"],
+            required: true 
+        },
```

Comment:
Do we really need to restrict possible Job Categories to these? Remember, another organization may be dealing in edTech but in a whole different field. Please think this through